vllm/scripts.py

```diff
@@ -125,6 +133,13 @@ def main():
     serve_parser.add_argument("model_tag",
                               type=str,
                               help="The model tag to serve")
+    serve_parser.add_argument("config",
```

Comment:
```suggestion
    serve_parser.add_argument("--config",
```

I think it's supposed to be an option instead of a positional argument?

Reply:
good catch! thank you..